src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+            RandomTeleportCommand.class,
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional)
+            )),
+    Backup(new ActionMenu(
+            "Backup",
+            "backup",
+            ForgePermissionNodes.Backup,
+            BackupCommand.class
+            )),
+    HomeTeleport(new ActionMenu(
+            "HomeTeleport",
+            "home",
+            ForgePermissionNodes.HomeTeleport,
+            HomeTeleportCommand.class
+            )),
+    RaidAllTonight(new ActionMenu(
+            "RaidAllTonight",
+            "raid-tonight",
+            ForgePermissionNodes.RaidAllTonight,
+            RaidAllTonightCommand.class
+            )),
+    RaidAllNow(new ActionMenu(
+            "RaidAllNow",
+            "raid-now",
+            ForgePermissionNodes.RaidAllNow,
+            RaidAllNowCommand.class
+            )),
+    CheckForAutoDeletes(new ActionMenu(
+            "CheckForAutoDeletes",
+            "check",
+            ForgePermissionNodes.CheckForAutoDeletes,
+            CheckForAutoDeletesCommand.class,
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional)
+            )),
+    WhoAmI(new ActionMenu(
+            "WhoAmI",
+            "whoami",
+            ForgePermissionNodes.WhoAmI,
+            WhoAmICommand.class
+            )),
+    WhereAmI(new ActionMenu(
+            "WhereAmI",
+            "whereami",
+            ForgePermissionNodes.WhereAmI,
+            WhereAmICommand.class
+            )),
+
+    ListColonies(new ActionMenu(
+            "ListColonies",
+            "list",
+            ForgePermissionNodes.ListColonies,
+            ListColoniesCommand.class,
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional),
+            new ActionArgument("abandonedSinceTimeInHours", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+            )),
+    RequestSystemResetAll(new ActionMenu(
+            "RequestSystemResetAll",
+            "rsResetAll",
+            ForgePermissionNodes.RequestSystemResetAll,
+            RSResetAllCommand.class
+            )),
+
+    ListCitizens(new ActionMenu(
+            "ListCitizens",
+            "list",
+            ForgePermissionNodes.ListCitizens,
+            ListCitizensCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+        )),
+    KillCitizen(new ActionMenu(
+            "KillCitizen",
+            "kill",
+            ForgePermissionNodes.KillCitizen,
+            KillCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "citizen" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)